sonic_platform_base/sfp_base.py

```diff
@@ -0,0 +1,204 @@
+        encoding                   |1*255VCHAR     |encoding information
+        ext_identifier             |1*255VCHAR     |extend identifier
+        ext_rateselect_compliance  |1*255VCHAR     |extended rateSelect compliance
+        cable_length               |INT            |cable length in m
+        mominal_bit_rate           |INT            |nominal bit rate by 100Mbs
+        specification_compliance   |1*255VCHAR     |specification compliance
+        vendor_date                |1*255VCHAR     |vendor date
+        vendor_oui                 |1*255VCHAR     |vendor OUI
+        ========================================================================
+        """
+        return NotImplementedError
+
+    def get_transceiver_bulk_status(self):
+        """
+        Retrieves transceiver bulk status of this SFP
+
+        Returns:
+            A dict which contains following keys/values :
+        ========================================================================
+        keys                       |Value Format   |Information	
+        ---------------------------|---------------|----------------------------
+        RX LOS                     |BOOLEAN        |rx lost-of-signal status,
+                                   |               |True if has rx los, False if not.
+        TX FAULT                   |BOOLEAN        |tx fault status,
+                                   |               |True if has tx fault, False if not.
+        Reset status               |BOOLEAN        |reset status,
+                                   |               |True if SFP in reset, False if not.
+        LP mode                    |BOOLEAN        |low power mode status,
+                                   |               |True in lp mode, False if not.
+        tx disable                 |BOOLEAN        |tx disable status,
+                                   |               |True tx disabled, False if not.
+        tx disabled channel        |HEX            |disabled TX channles in hex,
+                                   |               |bits 0 to 3 represent channel 0
+                                   |               |to channel 3.
+        ========================================================================
+        """
+        return NotImplementedError
+                
+    def get_reset_status(self):
+        """
+        Retrieves the reset status of SFP
+
+        Returns:
+            A Boolean, True if reset enabled, False if disabled
+        """
+        return NotImplementedError
+
+    def get_rx_los(self):
+        """
+        Retrieves the rx los (lost-of-signal) status of SFP
+
+        Returns:
+            A Boolean, True if SFP has rx los, False if not.
+            Note : rx los status is lached until a call to get_rx_los or a reset.
+        """
+        return NotImplementedError
+
+    def get_tx_fault(self):
+        """
+        Retrieves the tx fault status of SFP
```

Comment:
Capitalize "TX" in the comments of this function as well as in get_tx_disable() and get_tx_disable_channel() below.